more glossary improvements (#184)
OCR-D/ocrd-website: site/en/spec/glossary.md

### TextLine

A text line is a single row of [words](#word) within a text [region](#region). (Depending on the region's or page's orientation, and the script's writing direction, it can be horizontal or vertical.)

### Print space

From the [PAGE-XML content schema documentation](https://ocr-d.de/de/gt-guidelines/pagexml/pagecontent_xsd_Complex_Type_pc_PrintSpaceType.html)

> Determines the effective area on the paper of a printed page. Its size is equal for all pages of a book (exceptions: titlepage, multipage pictures).
>

> > It contains all living elements (except marginals) like body type, footnotes, headings, running titles.

Reviewer: ```suggestion
> It contains all living elements (except marginalia) like body type, footnotes, headings, running titles.
```

Reviewer: what is meant with "body type"?

Author: > what is meant with "body type"?

I don't know TBH. @tboenig this is your text, right? Does this come from [TEI/DTABf's body](http://deutsches-textarchiv.de/doku/basisformat/bodyAllg.html) by any chance?

Reviewer 2: I think @tboenig just wanted to avoid a recursive definition ("printspace contains printspace+marginalia"). Let's just call that "paragraphs and headings" for simplicity.

>

> > It does not contain pagenumber (if not part of running title), marginals, signature mark, preview words.

Reviewer: ```suggestion
> It does not contain pagenumber (if not part of running title), marginalia, signature mark, preview words.
```

has to be available at least locally. Ideally, a publicly available model repository will
be developed.

### Workspace

> A workspace is a transient repository for some document. It holds all physical data files required for run-time OCR-D processing, and may reference more data files in remote locations. Minimally a local filesystem copy of the [METS](https://ocr-d.de/en/spec/mets) file.

Reviewer 2: "repository" could be misleading (GT repo, LTA repo...), maybe "directory"?

Author: I know there's the VCS connotation to that word, but the entry adds to _software repository_, _ground truth repository_, _research data repository_ and _model repository_ – some of which are probably not going to be VCS repos either, will they? 

Moreover, I thought _transient repository_ would already disambiguate enough. Plus, we have always strived to parallel the repository metaphor in our workspace conceptualization and implementation. Why not keep the connotation here?

Author: > maybe "directory"?

It's not just the directory though, but filesystem structure plus METS...

Reviewer 3: I must agree that the term `repository` may rather confuse our target users here. "A workspace is a local directory that..." seems much clearer imho.

Author: @cneud I don't want to be picky (as I said above, I am ok with any changes), but on protocol, your comment plainly ignores the (arguments of the) 2 comments above it...

Reviewer 3: A `repository` can mean many things, a VCS is just one example. One may even argue that in the library and DH world,  a digital asset management system is typically the primary understanding. This is also the reason why we in OCR-D have used the term repository (imo too vaguely) for different "data locations", which is _already_ confusing users (I believe this was what @kba already [hinted at](https://github.com/OCR-D/ocrd-website/pull/184#discussion_r515061825)). In that sense my point remains that the connotation would only further add to that confusion here and should therefore rather be avoided.

Qualifying it further by _transient_ is imho not really helping either. Especially in glossary and documentation, we should strive to use terms that also non-expert users easily understand, whenever possible. Calling this a "workspace" is already an abstract concept for what it really is - "a file system structure plus METS" - which would be much more intuitive and easy to understand for most users than "transient repository".

Reviewer 3: This is completely independent from the fact that I personally like that connotation from a design standpoint ;)

Author: Thanks for elaborating! These are very convincing arguments IMO. You are right, we need maximum clarity here instead of more prose.

Then how about:

```suggestion
A workspace is a representation for some document in the local file system. Minimally it consists of a directory with a copy of the [METS](https://ocr-d.de/en/spec/mets) file. Additionally, that directory may contain physical data files and sub-directories belonging to the document (required or generated by run-time OCR-D processing), as referenced by the METS via `mets:file/mets:FLocat/@href` and `mets:fileGrp/@USE`. Files and sub-directories without reference (like log or config files) are not part of the workspace, as are references to remote locations. They can be added to the workspace by referencing them in the METS via their relative local path names.
```

Reviewer 3: Thank you, this spells it out exceptionally clear imo without the need for abstract concepts, perfect for the glossary!
